source/help/getting-started/organizing-your-sidebar.rst

```diff
@@ -0,0 +1,176 @@
+
+    .. image:: ../../images/multi-select-move.gif
+
+Muting and unmuting categories
+------------------------------
+
+When you mute or unmute a category, all channels within that category are also muted or unmuted. 
+
+Select the **Category options** icon in the sidebar, then select **Mute Category**.
+
+Once a category is muted:
+
+- Email, desktop, and push notifications are disabled for all channels in the category.
+- A mute icon displays next to each channel name in the category.
+- The category and all of its channels are greyed out in the left-hand sidebar. Channels in the category aren't marked as unread unless you’re mentioned directly.
+- You can selectively unmute specific channels within the category.
+
+To unmute the category, select the **Category options** icon in the sidebar, then select **Unmute Category**.
+
+    .. image:: ../../images/mute-categories.gif
+
+Sorting category channels
+-------------------------
+
+Using the Mattermost Web or Desktop App, you can control how channels are sorted for each category. Sort category channels manually (by dragging and dropping), alphabetically, or by recent conversations first.
+
+Select the **Category options** icon in the sidebar, then select **Sort** and choose from **Alphabetically**, **Recent Activity**, or **Manually**.
+
+    .. image:: ../../images/sort-categories.gif
+
+Grouping unread messages
+------------------------
+
+Catch up on all your unread channels in one place at the top of your sidebar with a one-click **Unreads** category.
+
+Go to **Main Menu > Account Settings > Sidebar**, set **Group unread channels separately** to **On**, then select **Save**.
+
+- When this setting is enabled, all unread messages appear only in the **Unreads** category.
+- When this setting is disabled, all unread messages appear within their respective categories and channels.
+
+When enabled, the **Unreads** category is organized as follows:
+
+- Unread messages that contain mentions are sorted to the top by most recent activity.
+- Unread messages that do not contain mentions are sorted by most recent and appear directly below unread messages containing mentions.
+- Muted channels that contain mentions are sorted by most recent and appear directly below unread messages without mentions.
+
+    .. image:: ../../images/unreads.gif
+
+.. tip::
+  If you prefer to see a decicated unread-only view in your sidebar, collapse all custom categories to show only unread messages, then disable **Grouping unread messages** under **Account Settings > Sidebar**.
+
+Sorting your Direct Messages
+----------------------------
+
+Sort your Direct Messages alphabetically or by recent conversations first.
+
+Select the **Channel options** icon in the sidebar, then select **Sort** and choose from **Alphabetically** or **Recent Activity**.
+
+How many Direct Messages to display?
```

Comment:
```suggestion
Controlling the number of displayed Direct Messages
```

Reply:
@justinegeffen A heading this long tends to get cut off in the LHS. Do you have suggestions for avoiding such cutoff?

Reply:
Good point! How about "Managing displayed Direct Messages"?